### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 1shin-7**
- Adds badge.
- Beautification. 
- Optimize Quick setup. 
- Use table to list commands. 
- add Linux compilation guide.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Looks great, but java 16 is now required on the development branch. Also, please put the building section a little higher in the readme (preferbably above the commands section)

**Comment by lunaticwhat** [comment]
nice work ! 
but development branch required different java development kit version and also have some different information. You can commit your edit later after @Melledy approved my commit #181

**Comment by Wansn-w** [comment]
> Looks great, but java 16 is now required on the development branch. Also, please put the building section a little higher in the readme (preferbably above the commands section)

Modified

File: README.md
Comment (lunaticwhat) [inline README.md]: actually changed into start.cmd now

**Review by lunaticwhat — CHANGES_REQUESTED** [review]

**Comment by 1shin-7** [comment]
I want to solve the conflict, but I've turned off the computer, so it's difficult to solve the conflict😢

**Comment by GoldElysium** [comment]
Hmm, maybe add another link to the wiki troubleshooting guide in the Quick Troubleshooting section?

**Comment by 1shin-7** [comment]
> Hmm, maybe add another link to the wiki troubleshooting guide in the Quick Troubleshooting section? 

That makes sense.... Btw I think it seems that the command should also be moved there, although I can't edit the wiki

**Comment by GoldElysium** [comment]
> > Hmm, maybe add another link to the wiki troubleshooting guide in the Quick Troubleshooting section?
> 
> That makes sense.... Btw I think it seems that the command should also be moved there, although I can't edit the wiki

Yup. Would make sense, would have to ask a maintainer to do that.

**Comment by lunaticwhat** [comment]
> I want to solve the conflict, but I've turned off the computer, so it's difficult to solve the conflict😢

uhh sometime just remove your fork and re-fork after that do the commit again xD easy way to solve conflicting

**Comment by 1shin-7** [comment]
> > I want to solve the conflict, but I've turned off the computer, so it's difficult to solve the conflict😢
> 
> uhh sometime just remove your fork and re-fork after that do the commit again xD easy way to solve conflicting 

i think so XD

**Pull request closed without merging**
